Modules/_winapi.c

```diff
@@ -1553,6 +1553,25 @@ _winapi_ReadFile_impl(PyObject *module, HANDLE handle, DWORD size,
     return Py_BuildValue("NI", buf, err);
 }
 
+/*[clinic input]
+_winapi.ResumeThread -> DWORD
+
+    handle: HANDLE
+    /
+[clinic start generated code]*/
+
+static DWORD
+_winapi_ResumeThread_impl(PyObject *module, HANDLE handle)
+/*[clinic end generated code: output=c1f717bebfc83593 input=6297b36b9bdebc19]*/
+{
+    DWORD result;
+    Py_BEGIN_ALLOW_THREADS
+    result = ResumeThread(handle);
+    Py_END_ALLOW_THREADS
```

Comment:
f you look at the generated clinic code, you'll see that it depends on an exception being set in order to make it return `NULL` when `ResumeThread` returns -1. You still need to set an exception for an error. For example:

```
DWORD result;
Py_BEGIN_ALLOW_THREADS
result = ResumeThread(handle);
Py_END_ALLOW_THREADS
if (result == -1)
    PyErr_SetFromWindowsErr(0);
return result;
```

Reply:
Fixed!